### PR TITLE
[tensorflow] Update TF2.5 images to only upgrade systemd

### PR DESCRIPTION
**Pull request by arjkesh**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Avoid upgrading every package, only upgrade systemd

### Tests run
Run PR tests

### DLC image/dockerfile
TF2.5 dockerfiles

### Additional context

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/inference/docker/2.5/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -29,7 +29,8 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # nginx + njs
 RUN apt-get update \
- && apt-get -y upgrade \
```
Comment (jeet4320) [inline tensorflow/inference/docker/2.5/py3/Dockerfile.cpu:32]: is running `apt-get -y upgrade` not updating `systemd`

Comment (arjkesh) [reply tensorflow/inference/docker/2.5/py3/Dockerfile.cpu:32]: It is, but it is also upgrading many packages under the hood. We want to maintain consistency in our build system as much as possible from build to build, so we're only updating packages we need to upgrade.

**Comment by shantanutrip** [comment]
I think we can get this merged. We have confirmed that the PR has the same issue that we see on the mainline. So, there is no need to keep the PR waiting. We can merge it and the failure can be taken care of in a totally different PR.

**Review by shantanutrip — APPROVED** [review]

**Comment by arjkesh** [comment]
TF2 sanity test failures are for unrelated reason (instance setup for telemetry test). I'm going to merge this in so that we have the correct image building. We'll also need to follow up on systemd issues for TF1

**Pull request merged**